adcircpy/server/slurm_config.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         Instantiate a new Slurm shell script (`*.job`).
 
         :param account: Slurm account name
-        :param slurm_ntasks: number of Slurm tasks
+        :param slurm_ntasks: number of total tasks
```

Comment:
Let's just call it ntasks ... this is the SlurmConfig class after all, so the slurm_ prefix is redundant

Reply:
good idea